pkg/manifests/manifests.go

```diff
@@ -455,9 +461,41 @@ func MachineConfig(component string) (*machineconfigv1.MachineConfig, error) {
 
 func getIgnitionConfig() ([]byte, error) {
```

Comment:
the core idea is correct, but this logic should not be done implicitely - and better not make calls querying the cluster state in this code. Let's just pass it the desired version as flag and let's get and pass the flag in the upper layers.

what about
```
func getInitionConfig(ver platform.Version) ([]byte, error) {
       selinuixPolicy := rteassets.GetSELinuxPolicy(ver)
       ...
}
```

Reply:
Yeah, that will certainly make the code nice and clean. Thanks for the input.